labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -15,15 +15,14 @@ object ListManipulationExercise01 {
    * 
    */
   def firstElementInList[T](l: List[T]): T = {
-    //buildin
-    null.asInstanceOf[T]
+    l(0)
   }
```

Comment:
It's interesting to see the different ways people wrote simple functions like this. I used the head method and didn't know until now that your solution was a syntactically valid way to get the beginning of a list.